src/Microsoft.ServiceFabric.Actors/Runtime/ActorBase.cs

```diff
@@ -161,14 +161,14 @@ protected virtual Task OnPreActorMethodAsync(ActorMethodContext actorMethodConte
         }
 
         /// <summary>
-        /// This method is invoked by actor runtime an actor method has finished execution. Override this method
-        /// for performing any actions after an actor method has finished execution. 
+        /// Overrides this method for performing any actions after an actor method has finished execution.
```

Comment:
@v-rboado  Should be "Override".  A developer would need to override this method.